src/effects/Equalization.cpp

```diff
@@ -414,6 +414,31 @@ bool EffectEqualization::TransferDataToWindow(const EffectSettings &settings)
    return mUI.TransferDataToWindow(settings);
 }
 
+namespace {
+struct EqualizationTask {
```

Comment:
I'd expect that entity called Task is a callable object or at least have `Run` or `Execute` or similar named method

Reply:
Wait for further developments... ;-)